global.json

```diff
@@ -0,0 +1,6 @@
+{
+  "sdk": {
+      "rollForward": "latestMajor",
+      "allowPrerelease": false
+  }
+}
```

Comment:
Why do you feel it is required to introduce `global.json`, what issue does it address ?

Reply:
It is a wish from Egil, so you can have something VS in prerelease

Reply:
YAGNI - This should only be there if we have an issues that can't be address by other means.
However - looking at other open-source projects they do tend to have a `global.json` in the root folder, so lets keep it.